.tmuxinator.yml

```diff
@@ -45,13 +45,11 @@ pre_window: esy shell
 windows:
   - editor:
       layout: tiled
-      # Synchronize all panes of this window, can be enabled before or after the pane commands run.
-      # 'before' represents legacy functionality and will be deprecated in a future release, in favour of 'after'
-      # synchronize: after
+      # Due to an esy bug, multiple `esy x`s can't run at the same time, so we use sleep to space them out
+      # Also, we need to add a delay to the first one to give the `on_project_start` command time to run
       panes:
-        - sleep 1 && $cur__target_dir/default/bin/deku_node.exe 0
-        - sleep 1 && $cur__target_dir/default/bin/deku_node.exe 1
-        - sleep 1 && $cur__target_dir/default/bin/deku_node.exe 2
-        - sleep 1 && $cur__target_dir/default/bin/deku_node.exe 3
-        - $cur__target_dir/default/bin/sidecli.exe make-credentials && sleep 3 && $cur__target_dir/default/bin/sidecli.exe inject-genesis
+        - sleep 1 && esy x deku-node 0
+        - sleep 3 && esy x deku-node 1
```

Comment:
```suggestion
        - sleep 2 && esy x deku-node 1
```

Reply:
In my testing a 1 second delay between nodes was enough. @anchpop if you can confirm this, then please accept all the suggestions. Otherwise, reject all.